app/controllers/users_controller.rb

```diff
@@ -19,8 +26,53 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update user_params
+      flash[:success] = t ".success_update"
+      redirect_to @user
+    else
+      render "edit"
```

Comment:
```suggestion
      render :edit
```
thêm flash nữa e nhé